test/functional/desktop-driver-e2e-specs.js

```diff
@@ -28,20 +28,23 @@ describe('Desktop SafariDriver', function () {
     }
   });
   beforeEach(async function () {
-    driver = wd.promiseChainRemote(HOST, PORT);
-    await driver.init(CAPS);
+    driver = await remote({
+      hostname: HOST,
+      port: PORT,
+      capabilities: CAPS,
+    });
   });
   afterEach(async function () {
     if (driver) {
-      await driver.quit();
+      await driver.deleteSession();
       driver = null;
     }
   });
 
   it('should start and stop a session', async function () {
-    await driver.get('https://appium.io/');
-    const button = await driver.elementByCss('#downloadLink');
-    await button.text().should.eventually.eql('Download Appium');
+    await driver.url('https://appium.io/');
+    const button = await driver.findElement('css', '#downloadLink');
```

Comment:
I think the preferred way to do this in wdio would be

```js
await driver.$('#downloadLink')
```

Reply:
I'm not an expert in wdio, so I fully trust your advices :)